usermods/Ble2Json_v2/usermod_v2_ble2json.h

```diff
@@ -0,0 +1,381 @@
+#pragma once
+
+#include "wled.h"
+// #include "ble.h"
```

Comment:
If the file is not needed (is included below) remove it (and references to it) from PR.

Reply:
thx... I think I cleaned up all commented out code, but I will double check